src/middleware/storeProtectLoadUtils.js

```diff
@@ -1,13 +1,13 @@
-import { isCmsUi } from '@plone/volto/helpers';
+import { isCmsUi } from "@plone/volto/helpers";
 
-const LOCATION_CHANGE = '@@router/LOCATION_CHANGE';
-const PROTECT_START = '@@loadProtector/START';
-const PROTECT_END = '@@loadProtector/END';
-const PROTECT_SKIPPED = '@@loadProtector/SKIPPED';
-const GET_CONTENT_PENDING = 'GET_CONTENT_PENDING';
-const GET_CONTENT_SUCCESS = 'GET_CONTENT_SUCCESS';
-const GET_CONTENT_FAIL = 'GET_CONTENT_FAIL';
-const RESET_CONTENT = 'RESET_CONTENT';
+const LOCATION_CHANGE = "@@router/LOCATION_CHANGE";
+const PROTECT_START = "@@loadProtector/START";
+const PROTECT_END = "@@loadProtector/END";
+const PROTECT_SKIPPED = "@@loadProtector/SKIPPED";
+const GET_CONTENT_PENDING = "GET_CONTENT_PENDING";
+const GET_CONTENT_SUCCESS = "GET_CONTENT_SUCCESS";
+const GET_CONTENT_FAIL = "GET_CONTENT_FAIL";
+const RESET_CONTENT = "RESET_CONTENT";
```

Comment:
Revert all the above formatting changes and configure your editor not to convert quotes.